### PR TITLE
Merge _ Against

### PR DESCRIPTION
**Pull request by FWmohsin**

### CODE REVIEW

File: lib/bot.coffee [hunk 1]
```diff
@@ -25,6 +26,8 @@ ScheduleReceiver = require('./schedule_receiver')
 DataActionReceiver = require('./data_action_receiver')
 HealthCheckReceiver = require('./health_check_receiver')
 
+LOOKER_BOT_CHANNEL_ID = 'C14KGPH38' # hard coded to #looker-bot
```
Comment (popmonkey) [inline lib/bot.coffee]: you probably want these to be externally kept configuration options

File: lib/bot.coffee [hunk 2]
```diff
@@ -235,10 +239,31 @@ controller.on "direct_message", (bot, message) ->
...
+    console.log(text)
+    defaultBot.send({
+      text: text
+      channel: LOOKER_BOT_CHANNEL_ID
+    })
+  console.log('message: ' + JSON.stringify(message, null, 2))
+  if message.type == 'slash_command'
+    sendLog(message.user_name, 'Slash Command', message.text)
+  else if message.type == 'message'
+    bot.api.users.info({user: "U047EMSKV"}, (err, resp) ->
```
Comment (popmonkey) [inline lib/bot.coffee]: you probably want the slack user id to be externally kept configuration options - not in a public github repo.

File: lib/bot.coffee (hunk 1 above)
Comment (FWmohsin) [inline lib/bot.coffee]: Not mine. I just forked it from Coursera.

File: lib/bot.coffee (hunk 2 above)
Comment (FWmohsin) [inline lib/bot.coffee]: Coursera handy work.

**Pull request closed without merging**